packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -450,7 +465,54 @@ To update this comment, add \`${REFRESH_LABEL}\` label.
   );
 }
 
+/**
+ * Creates a comment mark used for addOrupdateissuecomment.
+ * I'll move this function to gcf-utils later.
+ */
+function getCommentMark(installationId: string): string {
+  return `<!-- probot comment [${installationId}]-->`;
+}
+
 export = (app: Application) => {
+  app.on('issue_comment.edited', async context => {
+    const repoUrl = context.payload.repository.full_name;
+    const configOptions = await getConfigOptions(context);
+
+    if (configOptions === null) {
+      logger.info(`snippet-bot is not configured for ${repoUrl}.`);
+      return;
+    }
+    const configuration = new Configuration({
+      ...DEFAULT_CONFIGURATION,
+      ...configOptions,
+    });
+    logger.info({config: configuration});
+    const commentMark = getCommentMark(context.payload.installation.id);
+
+    // If the comment is made by bots, and the comment has the refresh
+    // checkbox checked, we'll proceed.
+    if (
+      !context.payload.comment.body.includes(commentMark) ||
+      !context.payload.comment.body.includes(REFRESH_STRING)
+    ) {
+      return;
+    }
```

Comment:
Move this to the beginning of this block? No need to fetch the config if the comment isn't right (which doesn't require a network call).

Reply:
Thanks, it's a good call, done.